LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Yapp, Inc and contributors
+Copyright (c) 2019 Yapp, Inc and contributors
```

Comment:
Maybe we can just remove this Copyright instead of updating it?
@lukemelia Do you mind? What do you think?

Reply:
@Alonski no objection here

Reply:
@vasind Can you do this tiny thing :)